src/TiciTacaToeyGameEngine.ts

```diff
@@ -209,6 +223,12 @@ class TiciTacaToeyGameEngine implements GameEngine {
             message.connection
           );
         }
+        if (message.allotedTime) timePerPlayer = message.allotedTime;
```

Comment:
As a convention, to add consistency, we do not write if conditions without `{ }`

Reply:
Also recommended to add a new line above and below if conditions and function declarations

Reply:
This is also incorrect because when the first game updates the value of timePerPlayer to 10000 all subsequent games for all other players will end up using this value.